neofsid/doc.go

```diff
@@ -19,3 +19,18 @@ contract.
 NeoFSID contract does not produce notifications to process.
 */
 package neofsid
+
+/*
+Contract storage model.
```

Comment:
what does that line mean?

Reply:
i don't know how to answer, it means itself 
u think its redundant/incorrect?

Reply:
if it is a key for the whole file (`doc.go`) then sometimes the lines below are not `only` about the storage model; if it is a title for some paragraph then there are titles that are not the same as i see (every further titles are `# ...`)

Reply:
the rest of the blocks are hints to the stored data, not a schema, but still part of the descriptive model. But I understand your position, it is also reasonable. If u dont mind let's leave it as it is, we can always change, it's cosmetic